server/src/http/feature_refresher.rs

```diff
@@ -299,6 +311,97 @@ impl FeatureRefresher {
+            // let broadcaster = self.broadcaster.clone();
+
+            tokio::spawn(async move {
+                let mut stream = es_client
+                    .stream()
+                    .map_ok(move |sse| {
+                        let token = token.clone();
+                        let refresher = refresher.clone();
+                        // let broadcaster = broadcaster.clone();
+                        async move {
+                            match sse {
+                                // The first time we're connecting to Unleash. Just store the data.
+                                eventsource_client::SSE::Event(event)
+                                    if event.event_type == "unleash-connected" =>
+                                {
+                                    debug!(
+                                        "Connected to unleash! I should populate my flag cache now.",
+                                    );
+
+                                    // very rough handling of client features.
+                                    let features: ClientFeatures =
+                                    serde_json::from_str(&event.data).unwrap();
+                                    refresher.handle_client_features_updated(TokenRefresh::new(token, None), features);
+                                }
+                                // Unleash has updated. This is where we send data to listeners.
+                                eventsource_client::SSE::Event(event)
+                                    if event.event_type == "unleash-updated" =>
+                                {
+                                    debug!(
+                                        "Got an unleash updated event. I should update my cache and notify listeners.",
+                                    );
+
+                                    // store the data locally
+                                    let features: ClientFeatures =
+                                    serde_json::from_str(&event.data).unwrap();
+                                    refresher.handle_client_features_updated(TokenRefresh::new(token, None), features);
+
+                                    // send the data to the broadcaster. This should probably just send the new
+                                    // feature set OR even just a "filter flags"
+                                    // function. The broadcaster will take care
+                                    // of filtering the flags per listener.
+                                    // let data = Data::new(event.data).event("unleash-updated");
+                                    // broadcaster.rebroadcast(actix_web_lab::sse::Event::Data(data)).await;
+                                    refresher.broadcaster.broadcast().await;
+                                }
+                                eventsource_client::SSE::Event(event) => {
+                                    debug!(
```

Comment:
let's bump this to info! and set rust log for Edge to info. (for monitoring in prod)